### ISSUE TITLE
so weird net architecture and loss_func

### ISSUE THREAD

**Issue by leeswan** [YeongHyeon/ConAD-PyTorch#1]

**Comment by YeongHyeon** [comment]
This repository shows conceptual implementation.
The original text and the network structure may be different, but the loss function is the same.
